dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/deletefiles/DeleteItemPresenter.java

```diff
@@ -134,27 +140,22 @@ private void closeDialog() {
     }
 
     private void constructMessageForLog() {
-        String message = null;
+        String message;
         String alertMessage = null;
 
         if (deleteTask instanceof Ds3DeleteBucketTask) {
             alertMessage = resourceBundle.getString("deleteBucketErr");
-            message = alertMessage + StringConstants.SPACE
-                    + ((Ds3DeleteBucketTask) deleteTask).getErrorMsg();
         } else if (deleteTask instanceof Ds3DeleteFolderTask) {
             alertMessage = resourceBundle.getString("folderDeleteFailed");
-            message = alertMessage + StringConstants.SPACE
-                    + ((Ds3DeleteFolderTask) deleteTask).getErrorMsg();
         } else if (deleteTask instanceof Ds3DeleteFilesTask) {
             alertMessage = resourceBundle.getString("deleteFailedError");
-            message = alertMessage + StringConstants.SPACE
-                    + ((Ds3DeleteFilesTask) deleteTask).getErrorMsg();
         }
+        message = alertMessage + StringConstants.SPACE + deleteTask.getErrorMsg();
 
+        LOG.error("Failed to delete selected item(s) ", message);
```

Comment:
This error message looks incorrect.  did you mean to add `{}` to the end of the string?